app/src/main/java/org/mozilla/fenix/compose/MessageCard.kt

```diff
@@ -0,0 +1,220 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Card
+import androidx.compose.material.Icon
+import androidx.compose.material.IconButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.mozilla.experiments.nimbus.StringHolder
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.button.PrimaryButton
+import org.mozilla.fenix.gleanplumb.Message
+import org.mozilla.fenix.nimbus.MessageData
+import org.mozilla.fenix.nimbus.StyleData
+import org.mozilla.fenix.theme.FirefoxTheme
+import org.mozilla.fenix.theme.Theme
+
+/**
+ * Message Card.
+ *
+ * @param [Message] that holds a representation of GleanPlum message from Nimbus.
+ * @param onClick Invoked when user clicks on the message card.
+ * @param onCloseButtonClick Invoked when user clicks on close button to remove message.
+ */
+@Suppress("LongMethod")
+@Composable
+fun MessageCard(
+    message: Message,
+    onClick: () -> Unit,
+    onCloseButtonClick: () -> Unit,
+) {
+    Card(
+        modifier = Modifier
+            .padding(16.dp)
```

Comment:
We need to remove the padding I think. 

| Before      | After |
| ----------- | ----------- |
<img width="465" alt="Screen Shot 2022-05-17 at 1 35 24 PM" src="https://user-images.githubusercontent.com/1190888/168875540-cdde1eff-c780-4f8d-99d4-9a5fcef45dd2.png">|<img width="465" alt="Screen Shot 2022-05-17 at 12 52 57 PM" src="https://user-images.githubusercontent.com/1190888/168875558-f28608d5-2e6f-4232-9cc4-c83d798e56a1.png">

Reply:
No, that will cause something like this 
<img width="430" alt="image" src="https://user-images.githubusercontent.com/29518411/169112354-ade2fbed-ab54-4599-a1c5-7ca921b502bf.png">

Reply:
Isn't that the Column padding all = 16dp on line 66?